examples/asyncio.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Companion code to the blog post "Integrating Kafka With Python
+# Asyncio Web Applications"
+# https://www.confluent.io/blog/[path-to-blog-post]
+
+# Example Siege [https://github.com/JoeDog/siege] test:
+# siege -c 400 -r 200 'http://localhost:8000/items1 POST {"name":"testuser"}'
+
+import asyncio
+import confluent_kafka
+from confluent_kafka import KafkaException
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from time import time
+from threading import Thread
+import uvicorn
+
+
+class AIOProducer:
```

Comment:
doc strings wouldnt hurt

Reply:
yep. this omission caused by writing fatigue...